lib/web_ui/lib/src/engine/canvaskit/vertices.dart

```diff
@@ -47,6 +47,9 @@ class CkVertices implements ui.Vertices {
     Int32List? colors,
     Uint16List? indices,
   }) {
+    if (positions.length % 2 != 0) {
```

Comment:
there is an `isOdd` method on `int`, btw!

Reply:
I changed that to `if (positions.length.isOdd) {`

The `%2 != 0` check matched the pre-existing checks within `painting.dart`. 
I did not go and change those pre-existing checks in `painting.dart`, they are still using `if (positions.length % 2 != 0) {`

Using `isOdd` is definitely more readable!

Do you have any idea what the difference is in the code that the dart compiler creates for `%2!=0` vs `isOdd` ?
I tried quickly take a look at https://godbolt.org/ but it did not have luck.

Reply:
I'd hope it's all inlined. I have a vague memory that the code generated for JavaScript is better, though...